examples/cifar10_resnet.py

```diff
@@ -23,15 +25,46 @@
 
 # Training params.
 batch_size = 32
-epochs = 100
+epochs = 180
 data_augmentation = True
 
+#           |      |           | Orig Paper|           | Orig Paper|
+# Model     |  n   | ResNet v1 | ResNet v1 | ResNet v2 | ResNet v2 | sec/epoch
+#           |      | %Accuracy | %Accuracy | %Accuracy | %Accuracy | GTX 1080Ti
+# ResNet20  |  3   | 91.95     | 91.25     | 92.57     | -         | 58
+# ResNet32  |  5   | 92.00     | 92.49     | 92.22     | -         | 96
+# ResNet44  |  7   | 91.07     | 92.83     | 91.02     | -         | 128
+# ResNet56  |  9   | 90.25     | 93.03     | 91.37     | -         | 163
+# ResNet110 |  18  | 90.23     | 93.39     | 91.22     | 93.63     | 330
+n = 3
+
+# Orig paper: version = 1 (ResNet v1), Improved ResNet: version = 2 (ResNet v2)
+version = 1
+
+# Subtracting pixel mean improves accuracy
+use_pix_mean = True
+
 # Network architecture params.
 num_classes = 10
-num_filters = 64
-num_blocks = 4
-num_sub_blocks = 2
-use_max_pool = False
+num_filters = 16
+num_blocks = 3
+num_sub_blocks = 2 * n
+
+
+# Learning rate scheduler - called every epoch as part of callbacks
+def lr_schedule(epoch):
```

Comment:
Such methods should have a docstring to explain what they do.

Reply:
Docstring added.